01_getting_started/hello_world.py

```diff
@@ -106,21 +103,20 @@ def main():
 # running code locally. That means no waiting for long image builds when you've just moved a comma,
 # no fiddling with container image pushes, and no context-switching to a web UI to inspect logs.
 #
-# ### You can map over more data
+# ### Map over more data
 #
 # Change the `map` range from `20` to some large number, like `1170`. You'll see
 # Modal create and run even more containers in parallel this time.
 #
 # And it'll happen lightning fast!
 #
-# ### You can run a more interesting function
+# ### Run something more interesting
 #
-# The function `f` is obviously silly and doesn't do much, but in its place
+# The function `f` is silly and doesn't do much, but in its place
 # imagine something that matters to you, like:
 #
-# * Running [language model inference](/docs/examples/vllm_mixtral) or [fine-tuning](/docs/examples/slack-finetune)
-# * Manipulating [audio](/docs/examples/discord-musicgen) or [images](stable_diffusion_xl_turbo)
+# * Deploying [language model inference](/docs/examples/vllm_mixtral) or [fine-tuning](/docs/examples/slack-finetune) workloads.
+# * Manipulating [audio](/docs/examples/discord-musicgen) or [images](/docs/examples/stable_diffusion_xl_turbo).
 # * [Collecting financial data](/docs/examples/fetch_stock_prices) to backtest a trading algorithm.
 #
-# Modal lets you parallelize that operation effortlessly by running hundreds or
-# thousands of containers in the cloud.
+# Modal lets you parallelize these operations effortlessly by running hundreds of containers in the cloud.
```

Comment:
Why drop the "thousands"?

Reply:
It's a bit wordy. Also you can't actually do this. :)